lib/reporters/Runner.js

```diff
@@ -16,7 +16,8 @@ define([
 		config = config || {};
 
 		this.sessions = {};
-		this.hasErrors = false;
+		this.hasFatalErrors = false;
+		this.hasSuiteErrors = false;
```

Comment:
Reporting suite errors as fatal errors was confusing. I changed it so that suite errors are actually reported as suite errors and fatal errors as fatal errors.